src/views/profile/edit.ejs

```diff
@@ -184,11 +185,10 @@
         <option value="KZ">Kazakhstan</option>
         <option value="KE">Kenya</option>
         <option value="KI">Kiribati</option>
-        <option value="KP">Korea, Democratic People's Republic of</option>
-        <option value="KR">Korea, Republic of</option>
+        <option value="XK">Kosovo</option>
```

Comment:
This looks new. Not sure if we have a flag icon for it?

Reply:
Ah. There is a new one. I'll get a flag when i can.

Reply:
Thanks!